### PR TITLE
[HCCDOC#1765]: Add recommendations-filtering content to Remote Health Monitoring

### PR DESCRIPTION
**Pull request by kblake2016**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/HCCDOC-1765

Link to docs preview:
Two sections added
- https://70364--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster#insights-operator-advisor-recommendation-filters_using-insights-to-identify-issues-with-your-cluster
- https://70364--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster#insights-operator-using-filters-on-advisor-recommendations_using-insights-to-identify-issues-with-your-cluster

QE review:
- [X ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kblake2016** [comment]
@jholecek-rh; because you provided the most feedback for this best-practice filtering content, please ack the QE review.

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:23]: We can make this procedure generic and simple, rather sepcifying any specific filter like Category.

. From the *filter types* list that is present above the list of recommendations, select a filter.
. Set the value of the filter by selecting the relevant options from the drop-down list.
. Optional: Apply additional filters by performing the same steps to further refine the recommendations list.

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:24]: We can remove this line to make this procedure generic.

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:11]: We can remove this sentence and try to make the below procedure generic, rather making it specific by applying only the Category filter.

File: modules/insights-operator-advisor-recommendation-filters.adoc
Comment (shipsing) [reply modules/insights-operator-advisor-recommendation-filters.adoc:29]: Can we simplify the description of the the `Likelihood` filter type in the above list?

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:7]: Rename this procedure module to `Applying a filter` to make it simple and clear.

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:27]: Simplify the 

```suggestion
* After applying filters, you can view the updated recommendations list. The applied filters are added next to the default filters.
```

File: modules/insights-operator-using-filters-on-advisor-recommendations.adoc
Comment (shipsing) [inline modules/insights-operator-using-filters-on-advisor-recommendations.adoc:32]:
```suggestion
You can apply multiple filters to the list of recommendations and remove them individually by performing one of the following actions:
* Click the clear icon next to the filter.
* Click *Reset filters* to return to the default view without any manually applied filters.
```
I do not see the *Remove filters* button in the GUI. Please check the second bullet point for correctness.

**Review by shipsing — CHANGES_REQUESTED** [review]
@kblake2016, I have provided a few suggestions for you to consider. Please have a look at them.
 
Since i am new to the peer review squad, i would ask @bburt-rh to give my review the go-ahead. Please wait until Brain applies the peer-review-done label.

**Comment by kblake2016** [comment]
@shipsing , thank you for the helpful feedback. I implemented it. I changed file names and titles/headings to simplify things. However, now I'm seeing a asciidoc validation error, which I think is due to changing one of the file names. Can you advise how to proceed?

File: support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
Comment (xJustin) [inline support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc:19]:
```suggestion
include::modules/filtering-insights-advisor-recommendations.adoc[leveloffset=+2]
```

**Review by xJustin — COMMENTED** [review]
Hey @kblake2016 I think I found your problem.

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/filtering-insights-advisor-recommendations.adoc:19]:
```suggestion
. From the [add label name!] drop-down list, select each category of recommendation you want to view.
```

- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements), "When you refer to the location of an element in a graphical user interface, do not use sensory words, such as _top_, _bottom_, _right_, or _left_." 
- Use [the ISG standards for drop-down lists](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list) whenever you refer to these UI elements.
- Per the ISG, do not use "check" to refer to marking a checkbox. Use "select".
- Also if it is a checkbox, refer to it as a "checkbox" and not a "box".

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/filtering-insights-advisor-recommendations.adoc:20]:
```suggestion
. Optional: Add additional filters to further refine the list.
```
Follow [the ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) for indicating optional steps.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:16]: Not sure about this, but maybe the items here would be easier to read at a glance if they were in a table rather than a bulleted list? WDYT?

In any case, per [the ISG guidance on the wording of list items](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), all of the first sentences of the descriptions must be made grammatically parallel. Re-write so that either they all begin with an imperative "Search for...", "Filter by.." to form complete sentences or they begin with a descriptive sentence fragment (as in the "Total risk" item).

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:18]:
```suggestion
* *Total risk:* An abstract value derived from the likelihood that the detected issue will negatively affect your infrastructure and the potential impact on operations. Select one or more values from *Critical*, *Important*, *Moderate*, and *Low*.
```
- remove unneeded comma before subordinating conjunction
- edit to reduce extra words.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:21]:
```suggestion
* *Category:* Filter by categories of recommendations. Select one or more values from *Service Availability*, *Performance*, *Fault Tolerance*, *Security*, and *Best Practice*.
```
Same comment as previously. In general, try to refer to these options in a consistent way for each item in the list.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:22]:
```suggestion
* *Status:* Set to only show *Enabled* recommendations by default. You can also set the filter to show disabled recommendations.
```
- The meaning of this sentence is not clear to me. Why include the second part of the sentence? I don't know the functionality, so I'm not sure what I'm missing here. Is the setting for showing disabled recommendations something extra?

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:23]:
```suggestion
* *Clusters impacted: Filter only those recommendations currently impacting one or more clusters. You can also show non-impacting recommendations. This setting is enabled by default.
```
- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes#general-text), do not use the construction "and/or".

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/removing-filters-from-insights-advisor-recommendations.adoc:9]:
```suggestion
You can apply multiple filters to the list of recommendations or remove them individually. You can also reset the filters.
```
- Edit for minimalism and to eliminate the awkward grammatical construction.

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/removing-filters-from-insights-advisor-recommendations.adoc:15]:
```suggestion
* Click *Reset filters* to remove all the filters that you applied and to leave the default filters in place.
```
- Rewrite as a step in a procedure.

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/removing-filters-from-insights-advisor-recommendations.adoc:12]:
```suggestion
* Where the applied filters are displayed, remove applied filters individually, including the default filters, by clicking the *X* icon next to the filter.
```
- Rewrite as a step in a procedure.
- Reword to use [the ISG guidance on referring to UI elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui). Not sure if this is a drop-down list but the description needs to use the standard UI element wording for whatever it is.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:19]:
```suggestion
* *Impact:* The potential impact to the continuity of cluster operations. Select one or more values from *Critical*, *High*, *Medium*, and *Low*.
```
- Not sure if this change is accurate, but the previous list item used this construction.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (bburt-rh) [inline modules/insights-advisor-recommendations-filters.adoc:20]:
```suggestion
* *Likelihood:* Filter by the potential for a negative impact to clusters if the recommendation comes to fruition. Select one or more values from *Critical*, *High*, *Medium*, or *Low*.
```
- Same comment as for the previous item. In general, try to refer to these options in a consistent way for each item in the list.

**Review by bburt-rh — COMMENTED** [review]
Thanks for this PR! I've made a number of changes and suggestions to bring the language of this PR into alignment with IBM style guidance and OpenShift docs guidelines.

Overall, I would suggest re-reviewing the [ISG guidance for referring to UI elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui) to ensure that the UI references in this PR are aligned with those standards.

CC; @shipsing

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:9]: Done

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:17]: Done

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:18]: Done

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:19]: Done. Just as an fyi, the dropdowns on the Hybrid Cloud Console filters don't have label names (or hover-over names); there are two of them, and they're side by side. We call them the primary and secondary filters in another Console project, but neither the engineers nor peer review like that phrasing so I'm trying to be specific for the users, given both the limitations and rapidly changing nature of the GUI.

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/removing-filters-from-insights-advisor-recommendations.adoc:9]: done

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/removing-filters-from-insights-advisor-recommendations.adoc:15]: done

File: modules/removing-filters-from-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/removing-filters-from-insights-advisor-recommendations.adoc:12]: It's not a drop-down list. Each selected filter looks like an ovaltangular chip with a tiny x in the upper-left corner of it. All of the selected filters are visible at once. To remove individual filters, you click the x on the filter "chip." To remove them all (except for the defaults), you click (the blue words that aren't a button) *Reset filters*.

I rewrote each of these as procedures, but still used bullets since they're single steps.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (kblake2016) [reply modules/insights-advisor-recommendations-filters.adoc:19]: Good change. The meaning isn't changed. Done.

File: modules/insights-advisor-recommendations-filters.adoc
Comment (kblake2016) [reply modules/insights-advisor-recommendations-filters.adoc:20]: Done

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:20]: Done

File: modules/insights-advisor-recommendations-filters.adoc
Comment (kblake2016) [reply modules/insights-advisor-recommendations-filters.adoc:16]: done

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/filtering-insights-advisor-recommendations.adoc:19]: I'm still wondering if there's some way to avoid referring to "left" and "right". [The IBM guidelines say not to refer to locations using sensory elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements). Do these lists have a label or a heading of some kind that you can refer to?

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (bburt-rh) [inline modules/filtering-insights-advisor-recommendations.adoc:17]:
```suggestion
. Go to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.
```
Current [IBM Style word usage guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) say to use _navigate_ "only when a more direct verb, such as _click_, _select_, _browse_, or _go to_, is not applicable".

**Review by bburt-rh — APPROVED** [review]
Looks great now, @kblake2016! Very clean. Thanks so much for getting in all those changes. I've added a couple of other small things for you to consider changing before sending it on for merge review, but I'm going to go ahead and give this the `peer-review-done` label.

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:19]: I've changed the first instance from "left filter drop-down list" to "main, filter-type drop-down list," and the second instance from "right drop-down list" to "filter-value drop-down list." It still sounds inelegant to me, but I'll take this to my team and ask how to make this better with respect to the ISG and your feedback.

File: modules/filtering-insights-advisor-recommendations.adoc
Comment (kblake2016) [reply modules/filtering-insights-advisor-recommendations.adoc:17]: Done

**Comment by kblake2016** [comment]
@bburt-rh , I've implemented your latest recommendations locally and updated the PR here. I'm preemptively requesting the merge review; I think that's the next step. Thanks again for your time and attention.

/label merge-review-needed

**Pull request merged**
